packages/devops/scripts/deployment-vm-db/setup.sh

```diff
@@ -6,6 +6,7 @@ SCRIPT_DIR=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
 cd "$SCRIPT_DIR"
 
 # install postgres 13
+# do not trigger if no psql
```

Comment:
Discussed, lets get rid the if check, it is safe to re-run these commands every time